docs/supported-technologies.asciidoc

```diff
@@ -96,6 +96,9 @@ Other Servlet 3+ compliant servers will most likely work as well.
 |<<setup-wildfly,WildFly>>
 |8-15
 
+|<<setup-wildfly,JBoss EAP>>
```

Comment:
links to "setup-wildfly"

Reply:
The setup instructions are the same. I'll modify the headline of `setup-wildfly` so that it's clear that it's for both.

Reply:
See https://github.com/elastic/apm-agent-java/pull/488

Reply:
thanks!